src/foundry/foundry.js/userTargets.d.ts

```diff
@@ -1,36 +1,38 @@
-/**
- * A subclass of Set which manages the Token ids which the User has targeted
- * @internal
- */
-declare class UserTargets extends Set<Token> {
-  constructor(user: UserTargets['user']);
-
-  user: User;
+import { ConfiguredDocumentClass } from '../../types/helperTypes';
 
+declare global {
   /**
-   * Return the Token IDs which are user targets
+   * A subclass of Set which manages the Token ids which the User has targeted.
+   * @see User#targets
    */
-  get ids(): string[];
+  class UserTargets extends Set<Token> {
```

Comment:
We could consider to use the configured token object class here and in other places. It's not yet configurable but we already have the helper type `ObjectClass` anyways...